src/Providers/PaymentProvider/CivicaPayProvider.cs

```diff
@@ -58,7 +60,15 @@ public async Task<string> GeneratePaymentUrl(string form, string path, string re
                             CallingAppTranReference = reference,
                             ServicePayItemDesc = paymentInformation.Settings.Description
                         },
-                        AddressDetails = new AddressDetail()
+                        AddressDetails = new AddressDetail
+                        {
+                            HouseNo = address.HouseNo,
```

Comment:
Using Pest Control as an example we only set HouseNo, Street, Town, and Postcode